app/admin.js

```diff
@@ -34,6 +33,15 @@ importKeys.addArgument(
     help: 'path to a list of public keys generated from admin.js generate'
   }
 );
+importKeys.addArgument(
+  ['--type'],
+  {
+    action: 'store',
+    help: 'type of key to import (xpub for most coins, xlm for Stellar keys)',
+    defaultValue: 'xpub',
+    choices: ['xpub', 'xlm']
```

Comment:
does xpub vs xlm make sense for this switch? Seems like the difference is more EC-specific than coin-specific, though having to type `--type ed25519` is unwieldy. Maybe 'hd' vs 'flat'?

Reply:
Ed25519 isn't quite an accurate name. Although multiple coins use ed25519 signing, they don't all serialize keys in the same format. The definition of a key as [G/S][32 bytes encoded as base32][checksum encoded as base32] is unique to Stellar.